mempool/clist_mempool.go

```diff
@@ -680,11 +688,6 @@ func (mem *CListMempool) recheckTxs() {
 	for e := mem.txs.Front(); e != nil; e = e.Next() {
 		memTx := e.Value.(*mempoolTx)
 
```

Comment:
Any reason you created a whole new loop to remove the `MsgPlaceOrder` and `MsgCancelOrder` transactions, instead of just doing this?

```suggestion
		// Don't recheck for PlaceOrder/CancelOrder transactions
		if mem.isClobOrderTransaction(memTx) {
		        mem.removeTx(memTx.tx, e, false)
			continue
		}
```